devtools/gradle/src/main/java/io/quarkus/gradle/QuarkusPlugin.java

```diff
@@ -69,9 +72,15 @@ private void registerTasks(Project project) {
 
         Task quarkusBuild = tasks.create(QUARKUS_BUILD_TASK_NAME, QuarkusBuild.class);
         Task quarkusDev = tasks.create(QUARKUS_DEV_TASK_NAME, QuarkusDev.class);
-        Task buildNative = tasks.create(BUILD_NATIVE_TASK_NAME, QuarkusNative.class);
         Task quarkusTestConfig = tasks.create(QUARKUS_TEST_CONFIG_TASK_NAME, QuarkusTestConfig.class);
 
+        Task buildNative = tasks.create(BUILD_NATIVE_TASK_NAME, DefaultTask.class);
+        buildNative.finalizedBy(quarkusBuild);
+        buildNative.doFirst(t -> project.getLogger()
+                .warn("buildNative task has been deprecated in favor of quarkusBuild -Dquarkus.package.type=native"));
```

Comment:
```suggestion
                .warn("The 'buildNative' task has been deprecated in favor of 'build -Dquarkus.package.type=native'"));
```

Reply:
Yes, this is better, I updated the warning